Sources/MapLibreSwiftUI/Models/MapCamera/CameraState.swift

```diff
@@ -26,7 +26,7 @@ public enum CameraState: Hashable {
     case trackingUserLocationWithCourse
     
     /// Centered on a bounding box/rectangle.
-    case rect(northeast: CLLocationCoordinate2D, southwest: CLLocationCoordinate2D) // TODO: make a bounding box?
+	case rect(boundingBox: MLNCoordinateBounds, edgePadding: UIEdgeInsets = .init(top: 20, left: 20, bottom: 20, right: 20))
```

Comment:
Once we decide on the right option for padding discussed in my large comment below, we can update this and put the defaults in the `Defaults` struct.